src/rdkafka_conf.c

```diff
@@ -850,12 +850,16 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           "belong to the same group." },
         { _RK_GLOBAL|_RK_CGRP|_RK_MED, "group.instance.id", _RK_C_STR,
           _RK(group_instance_id),
-          "Enable static group membership. "
+          "Enables static group membership. "
           "Static group members are able to leave and rejoin a group "
           "within the configured `session.timeout.ms` without prompting a "
-          "group rebalance. Each member of the group must have a "
+          "group rebalance. In order to take advantage of this new behavior "
+          "`session.timeout.ms` should be raised to a larger value. How large "
+          "you set this value will largely be contingent on your orchestration "
```

Comment:
Don't beat about the bush, this is too vague. Tell them what the timeout should be set to.
E.g., the maximum allowed time for a group member application to restart without triggering a rebalance, or whatever it is.

We're not using "you" form in any other configuration properties, formalize.